src/harps/io.jl

```diff
@@ -0,0 +1,171 @@
+"""
+   IO functions for the HARPS spectrograph
+    https://www.eso.org/sci/facilities/lasilla/instruments/harps.html
+Author: Alex Wise and collaborators
+Created: April 2021
+"""
+
+"""Create Dataframe containing filenames and key data for all files YYYY-MM-DDThh:mm:ss.mss (year-month-day T hour:minute:second:milisecond in directory"""
+function make_manifest(data_path::String ; max_spectra_to_use::Int = 1000 )
+    df_filenames = EchelleInstruments.make_manifest(data_path,r"^20\d{2}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}$")
+
+    df_files = DataFrame(read_metadata(joinpath(df_filenames.Filename[1],"e2ds.fits")))
+    keys = propertynames(df_files)
+    allowmissing!(df_files, keys[map(k->k∉[:Filename, :bjd, :target, :airmass],keys)] )
+
+    if length(df_filenames.Filename) >= 2
+        map(fn->add_metadata_from_fits!(df_files,joinpath(fn,"e2ds.fits")),df_filenames.Filename[2:end])
+    end
+
+    if all(df_files[:,:target] .== "HD128621")
```

Comment:
What a hardwired target name?

Reply:
if df_files[:,:target] all the same and file_exists(target_binaryRV.csv)
then load and print a message saying you're loading the binaryRVs

Reply:
add comment: TODO: check times match